server/logging/firehose.go

```diff
@@ -0,0 +1,141 @@
+package logging
+
+import (
+	"encoding/json"
+	"fmt"
+	"math"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/firehose"
+	"github.com/aws/aws-sdk-go/service/firehose/firehoseiface"
+	"github.com/pkg/errors"
+)
+
+const (
+	firehoseMaxRetries = 3
```

Comment:
Seems weird to me that you have this exponential back-off strategy but only try 3 times. Any reason why this is so low?

Reply:
agreed/